src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -307,6 +307,20 @@ private static bool IsFloatPointExpressionOfUnknownPrecision(BoundExpression rew
 
                     break;
 
+                case ConversionKind.MethodGroup:
+
+                    if (_inExpressionLambda || explicitCastInCode || (object)symbolOpt == null || !symbolOpt.IsStatic || symbolOpt.MethodKind != MethodKind.Ordinary)
+                    {
+                        break;
+                    }
+                    if (symbolOpt.IsExtensionMethod && ((BoundMethodGroup)rewrittenOperand).ReceiverOpt.Kind != BoundKind.TypeExpression)
+                    {
+                        break;
+                    }
+
+                    return RewriteMethodGroupConversion(oldNode, syntax, rewrittenOperand, symbolOpt, isExtensionMethod, rewrittenType);
```

Comment:
Do we have a unit-test that covers scenario with an extension method used as a normal static method?